templates/test/core/surface/public_headers_must_be_c89.c.template

```diff
@@ -25,13 +25,15 @@
       if platform_identifier in hdr:
         return True
     return False
+  def is_cronet_header(hdr):
+    return "cronet" in hdr
```

Comment:
Would it better be case insensitive?

Reply:
Since this is for the public C header files which should be in lower case, I think case making it case insensitive is not necessary.